src/lib_c/x86_64-windows-msvc/c/time.cr

```diff
@@ -0,0 +1,33 @@
+lib LibC
+  struct Filetime
+    low_date_time : DWord
+    high_date_time : DWord
+  end
+
+  struct SystemTime
+    year : Word
+    month : Word
+    day_of_week : Word
+    day : Word
+    hour : Word
+    minute : Word
+    second : Word
+    milliseconds : Word
+  end
+
+  struct TimeZoneInformation
+    bias : Long
+    standard_name : StaticArray(WChar, 32)
+    standard_date : SystemTime
+    standard_bias : Long
+    daylight_name : StaticArray(WChar, 32)
+    daylight_date : SystemTime
+    daylight_bias : Long
+  end
+
+  fun get_time_zone_information = GetTimeZoneInformation(tz_info : TimeZoneInformation*) : DWord
```

Comment:
The standard for functions (which I argued against) was to prefix them with an `_` so that their case could be kept the same. See `winapi.cr`.

Reply:
Okay, I didn't know that. I'll change it, allthough it seems more convenient to "crystalize" internal names. But there are pros and cons for both styles.

Reply:
You don't need the `_` anymore, just use `GetTimeZoneInformation`.

Opinion is to match syscalls 1:1. We don't have to go through crystal-like namings: they're not crystal methods, but Windows functions. Windows uses `GetTimeZoneInformation` not `get_time_zone_information`, let's respect that. It also makes automatically generating bindings (precomputed or on-the-fly) easier, and cleans up bindings by avoiding useless aliases.